digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/service/pointasset/DirectionalTrafficSignService.scala

```diff
@@ -41,9 +41,9 @@ class DirectionalTrafficSignService(val roadLinkService: RoadLinkService) extend
     }
   }
 
-  override def update(id: Long, updatedAsset: IncomingDirectionalTrafficSign, geometry: Seq[Point], municipality: Int, username: String, linkSource: LinkGeomSource): Long = {
+  override def update(id: Long, updatedAsset: IncomingDirectionalTrafficSign, roadLink: RoadLink, username: String): Long = {
     withDynTransaction {
-      updateWithoutTransaction(id, updatedAsset, geometry, municipality, username, linkSource)
+      updateWithoutTransaction(id, updatedAsset, roadLink.geometry, roadLink.municipalityCode, username, roadLink.linkSource)
```

Comment:
instead of , ...**roadLink.geometry, roadLink.municipalityCode,** username, **roadLink.linkSource** ...we could only as a parameter **roadLink.**